### PR TITLE
chore(docs): update exact notes on background ISR

### PR DESCRIPTION
**Pull request by j-mendez**
* ISR re-generating after the first request may not occur on new versions of node above 16.0.0 or depending on OS.
 * re-generating works when deploying on Vercel guaranteed.

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have helpful link attached, see `contributing.md`

## Documentation / Examples

- [ ] Make sure the linting passes by running `yarn lint`

### CODE REVIEW

File: docs/basic-features/data-fetching/incremental-static-regeneration.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ When a request is made to a page that was pre-rendered at build time, it will in
 
 - Any requests to the page after the initial request and before 10 seconds are also cached and instantaneous.
 - After the 10-second window, the next request will still show the cached (stale) page
-- Next.js triggers a regeneration of the page in the background.
+- Next.js triggers a regeneration of the page in the background. (May not work depending on your OS for background handling - works when deploying on Vercel)
```
Comment (ijjk) [inline docs/basic-features/data-fetching/incremental-static-regeneration.md:83]: This should work regardless of your OS, can you provide additional details where you are seeing this not work?

Comment (j-mendez) [reply docs/basic-features/data-fetching/incremental-static-regeneration.md:83]: Yes, using this in docker and the image is `mhart/alpine-node:16.2.0`. The first request can call getStaticProps when getStaticPaths is set to `fallback`. With `revalidate` the headers are pass properly for SWR but, the server does not handle the background step after it already has the cached content to invalidate.

Comment (ijjk) [reply docs/basic-features/data-fetching/incremental-static-regeneration.md:83]: Can you create a minimal reproduction on a new issue so we can investigate this? This should work fine inside of docker or any other OS so I don't think we want to document this and instead want to ensure it's working regardless of the env.

Comment (j-mendez) [reply docs/basic-features/data-fetching/incremental-static-regeneration.md:83]: Yes, I can get to this during the weekend np.

Comment (ijjk) [reply docs/basic-features/data-fetching/incremental-static-regeneration.md:83]: Great, I'm gonna close this in favor of an issue instead.

**Pull request closed without merging**